### DISCUSSION TITLE
5.0.1: Super admin doesn't have all admin rights

### DISCUSSION THREAD

**Discussion by oliversum-ch** [crater-invoice-inc/crater#657] [Q&A]
Hi! 
I created a second user with super admin rights. Nonetheless, this user doesn't have the rights to add new users, update the app etc.
Is this a bug or on purpose?
Thanks!

**Comment by mohitpanjwani** [comment]
Hey @oliversum-ch, its by design. 

Currently Crater only allows managing users, companies and other important configuration only from one account (i.e Team Owner) 

We're working on documenting this.
